graylog2-server/src/main/java/org/graylog/events/processor/aggregation/PivotAggregationSearch.java

```diff
@@ -144,9 +149,18 @@ public AggregationResult doSearch() throws EventProcessorException {
             });
 
             // If we have only EmptyParameterErrors, just return an empty Result
-            if (! (errors.stream().filter(e -> !(e instanceof EmptyParameterError)).count() > 1)) {
+            if (errors.stream().allMatch(e -> e instanceof EmptyParameterError)) {
                 return AggregationResult.empty();
             }
+
+            Notification systemNotification = notificationService.buildNow()
+                    .addType(Notification.Type.GENERIC)
```

Comment:
If we do this, we should introduce a new type, so it does not prevent other GENERIC notifications to show up.
Also, I'd use the new notification key with the event definition id.

Reply:
Agreed. That is an inconvenient limitation we will need to eliminate sometime.